test/functional/p2p_permissions.py

```diff
@@ -57,11 +59,13 @@ def run_test(self):
         # by modifying the configuration file.
         ip_port = "127.0.0.1:{}".format(p2p_port(1))
         self.nodes[1].replace_in_config([("bind=127.0.0.1", "whitebind=bloomfilter,forcerelay@" + ip_port)])
+        append_config(self.nodes[1].datadir_path, [f"bind=127.0.0.1:{tor_port(self.nodes[1].index)}=onion"])
```

Comment:
nit: adding a comment explaining the reason the line was added could help prevent regression in the future.

e.g.
```
# explicitly bind the tor port to prevent collision with the default port 
```

Reply:
Done